lib/components/narrative/metro/default-route-renderer.tsx

```diff
@@ -4,22 +4,28 @@ import styled from 'styled-components'
 
 // TODO: This should not be a section, but using div/span
 // doesn't allow us to style it from within the RouteBlock
-const Block = styled.section<{ color: string }>`
+const Block = styled.section<{ color: string; onColoredBackground?: boolean }>`
   background: #${(props) => props.color}1A;
-  padding: 3px 7px;
-  border-top: 5px solid #${(props) => props.color};
   border-radius: 5px;
-  text-align: center;
+  border-top: 5px solid #${(props) => props.color};
   display: inline-block;
   max-width: 12vw;
+  padding: 3px 7px;
+  text-align: center;
+  white-space: break-spaces;
+
+  ${(props) => props.onColoredBackground && 'border-top: none'}
```

Comment:
I agree that these extra props are messy, but I don't think either of us have an ideal solution here.

I think having the route renderer accept a `leg` may have been a bit ambitious. A future refactor that might be good is to have the route renderer accept only route name and color. This should be enough to still do the custom styling we're currently doing, and it would make this `onColoredBackground` prop less messy.